portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -30,18 +30,27 @@
   private final ArrayList<String> messages =  new ArrayList<>();
   private final Gson gson = new Gson();
 
-  @Override
-  public void init() {
-    messages.add("It would only take one hour to drive to space.");
-    messages.add("Google's first office was a rented garage.");
-    messages.add("Walmart has lower acceptance rate than Harvard.");
-  }
-
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("application/json");
     response.setCharacterEncoding("UTF-8");
     response.getWriter().print(gson.toJson(messages));
     response.getWriter().flush();
   }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String text = getParameter(request, "text-comment", "");
```

Comment:
Should you check if the text is empty, and not add it to the list?

Reply:
Doesn't the getParameter function do that?
Or should I create an if statement to not add it to the list?

Reply:
OH nvm I see the mistake